ui/src/testing/stubs/cohort-review-service-stub.ts

```diff
@@ -107,9 +105,7 @@ const cohortChartDataStub = {
 
 export class CohortReviewServiceStub extends CohortReviewApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.